### DISCUSSION TITLE
avrdude-v7.2-windows-x64 won't launch

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [avrdudes/avrdude#1611] [Q&A]
Rapidly blinks on the taskbar, then it's gone. running through avrdude.exe.

**Comment by stefanrueger** [comment]
It's a command-line tool meant to be started with options from the shell. See [documentation page 23](https://avrdudes.github.io/avrdude/7.2/avrdude.pdf) for example command line invocations.

**Comment by stefanrueger** [comment]
Closing this issue as we haven't heard from @DuckersMcQuack about more [details](https://github.com/avrdudes/avrdude/wiki/FAQ#how-to-report-an-issue-to-avrdude-project) of their experience.
